mock_states/status.commit1-02-failed.json

```diff
@@ -1,24 +1,31 @@
 {
   "pipeline_statuses": {
-    "buildkite/pipeline2/failed-step": {
+    "pipeline2": {
```

Comment:
not sure it's a good idea to lose the `buildkite/` prefix from the state, it is useful to know that it is a BK pipeline (and imagine in the future the monorepo has two providers with pipelines both named `monorepo`)

Reply:
The intent was to separate the pipeline name from the provider context, but it is more future proof to keep it, indeed. I will add this change to the backlog, as I have other tasks that need to be more prioritary at the moment.